orangecontrib/spectroscopy/widgets/owhyper.py

```diff
@@ -1068,7 +1202,14 @@ def output_image_selection(self):
     def init_attr_values(self, data):
         domain = data.domain if data is not None else None
         self.feature_value_model.set_domain(domain)
+        self.rgb_value_model.set_domain(domain)
         self.attr_value = self.feature_value_model[0] if self.feature_value_model else None
+        if self.rgb_value_model and len(self.rgb_value_model) >= 3:
```

Comment:
There is still a bug here: if a list (with the separators is >= 3, but < 3 without them, this will crash.